layers/+lang/ipython-notebook/packages.el

```diff
@@ -15,47 +15,66 @@
   (use-package ein
     :init
     (spacemacs/set-leader-keys "ayl" 'ein:login
-                               "ayr" 'ein:run
-                               "ays" 'ein:stop)
+      "ayr" 'ein:run
+      "ays" 'ein:stop)
     (spacemacs/declare-prefix "ay" "ipython notebook")
     :config
-    (require 'seq)
-    (require 'subr-x)
     (mapc
      (lambda (mode)
        (evil-define-minor-mode-key
-        mode 'ein:notebook-mode
-        (kbd "<C-return>") 'ein:worksheet-execute-cell-km
-        (kbd "<S-return>") 'ein:worksheet-execute-cell-and-goto-next-km))
+         mode 'ein:notebook
+         (kbd "<C-return>") 'ein:worksheet-execute-cell-km
+         (kbd "<S-return>") 'ein:worksheet-execute-cell-and-goto-next-km))
      '(insert hybrid normal))
     (evil-define-minor-mode-key
-     'normal 'ein:notebook-mode
-     "gj" 'ein:worksheet-goto-next-input-km
-     "gk" 'ein:worksheet-goto-prev-input-km)
+      'normal 'ein:notebook
+      "gj" 'ein:worksheet-goto-next-input-km
+      "gk" 'ein:worksheet-goto-prev-input-km)
     (with-eval-after-load 'ein-notebook
-      (evil-define-key nil ein:notebooklist-mode-map "o" 'spacemacs/ace-buffer-links)
-      (let ((bindings '(("j" ein:worksheet-goto-next-input-km)
-                        ("k" ein:worksheet-goto-prev-input-km)
-                        ("J" ein:worksheet-move-cell-down-km)
-                        ("K" ein:worksheet-move-cell-up-km)
-                        ("e" ein:worksheet-toggle-output-km)
-                        ("d" ein:worksheet-kill-cell-km)
-                        ("y" ein:worksheet-copy-cell-km)
-                        ("p" ein:worksheet-yank-cell-km)
-                        ("m" ein:worksheet-merge-cell-km)
-                        ("s" ein:worksheet-split-cell-at-point-km)
-                        ("o" ein:worksheet-insert-cell-below-km)
-                        ("O" ein:worksheet-insert-cell-above-km)
-                        ("t" ein:worksheet-toggle-cell-type-km)
-                        ("RET" ein:worksheet-execute-cell-km)
-                        ("l" ein:worksheet-clear-output-km)
-                        ("L" ein:worksheet-clear-all-output-km)
-                        ("C-s" ein:notebook-save-notebook-command-km)
-                        ("C-r" ein:notebook-rename-command-km)
-                        ("x" ein:notebook-close-km)
-                        ("z" ein:notebook-kernel-interrupt-command-km))))
+      (let ((bindings '(("y" ein:worksheet-copy-cell)
```

Comment:
One mustn't remove the `-km` suffixes.  Those are important.

I don't know who "tkamat" is, but many of the commands like "ein:notebook-worksheet-open-prev-or-last", "ein:notebook-worksheet-move-prev", and "ein:notebook-worksheet-open-1th" are suspect, and upstream has deprecated.

Reply:
>One mustn't remove the -km suffixes. Those are important.

Wow, it does solve the problem for me. Adding the -km bindings and being consistent in using ein:notebook-mode for the four cases, seems to solve it (I don't have neither the extra fuction in the minor-mode-map-alist, the emacs keybindings work well, and so does the evil keybindings!) 

>I don't know who "tkamat" is,

A user repository I found through Google :p 

> but many of the commands like "ein:notebook-worksheet-open-prev-or-last", "ein:notebook-worksheet-move-prev", and "ein:notebook-worksheet-open-1th" are suspect, and upstream has deprecated

It's a shame they were deprecated, It does seem convenient to be able to open several worksheets